forte/proc/dsrg.py

```diff
@@ -245,6 +245,26 @@ def compute_energy(self):
 
         return e_dsrg if len(self.energies) == 0 else e_relax
 
+    def compute_gradient(self, coupling_coefficients, ci_vectors):
+        """ Compute DSRG-MRPT2 analytic gradients. """
+        # Notes (Shuhe Wang):
+
+        e_relax = 0.0  # initialize this to avoid PyCharm warning
+
+        # Perform the initial un-relaxed DSRG
+        self.make_dsrg_solver()
+        self.dsrg_setup()
+        e_dsrg = self.dsrg_solver.compute_energy()
```

Comment:
I think this function should only compute gradient but not energy. 
```suggestion
```

Reply:
I agree. But yesterday when I tried to remove this line, the code failed.

Reply:
Like the T2_, need be passed from dsrg_mrpt2.cc

Reply:
This should be done outside but not here.

Reply:
solved